rootfs/api/middleware.py

```diff
@@ -46,5 +46,7 @@ def process_response(self, request, response):
         a response header.
         """
         # clients shouldn't care about the patch release
-        response['DEIS_API_VERSION'] = __version__.rsplit('.', 1)[0]
+        version = __version__.rsplit('.', 1)[0]
+        response['DEIS_API_VERSION'] = version
+        response['DEIS_PLATFORM_VERSION'] = version
```

Comment:
I know we did do that before but that's not always the best reason to keep doing it either :-)

I'll probably just do the full version then and hide it behind auth another time